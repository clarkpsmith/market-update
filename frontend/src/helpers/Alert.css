.Alert-text {
  color: rgb(235, 70, 70);
  font-size: 16px;
}
