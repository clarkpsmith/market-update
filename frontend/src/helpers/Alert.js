import React from "react";
import "./Alert.css";

const Alert = ({ messages }) => {
  const errorMessages = messages.map((err) => <div key={err}>{err}</div>);

  return <div className="Alert-text">{errorMessages}</div>;
};

export default Alert;
