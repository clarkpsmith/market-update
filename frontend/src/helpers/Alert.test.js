import React from "react";
import { render } from "@testing-library/react";
import Alert from "./Alert";

const messages = ["Invalid Username/Password"];

it("renders without crashing", () => {
  render(<Alert messages={messages} />);
});

it("matches snapshot", () => {
  const { asFragment } = render(<Alert messages={messages} />);
  expect(asFragment()).toMatchSnapshot();
});
