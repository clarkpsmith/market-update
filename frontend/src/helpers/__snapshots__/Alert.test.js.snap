// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Alert-text"
  >
    <div>
      Invalid Username/Password
    </div>
  </div>
</DocumentFragment>
`;
